docs/marionette.layoutview.md

```diff
@@ -105,73 +105,71 @@ new Marionette.LayoutView({
 
 ### LayoutView childEvents
 
-You can specify a `childEvents` hash or method which allows you to capture all
-bubbling `childEvents` without having to manually set bindings.
-
-The keys of the hash can either be a function or a string
-that is the name of a method on the layout view.
-
-The function is called in the context of the view. The first parameter is
-the child view, which emitted the event, the remainder are the arguments
-associated with the event.
+A `childEvents` hash or method permits handling of child view events without manually setting bindings. The values of the hash can either be a function or a string method name on the collection view.
 
 ```js
 // childEvents can be specified as a hash...
-var MyLayoutView = Marionette.LayoutView.extend({
+var MyCollectionView = Marionette.CollectionView.extend({
 
-  // This callback will be called whenever a child is rendered or emits a `render` event
   childEvents: {
-    render: function(childView) {
-      console.log("a childView has been rendered");
+    // This callback will be called whenever a child is rendered or emits a `render` event
+    render: function() {
+      console.log('A child view has been rendered.');
     }
   }
 });
 
 // ...or as a function that returns a hash.
-var MyLayoutView = Marionette.LayoutView.extend({
+var MyCollectionView = Marionette.CollectionView.extend({
 
   childEvents: function() {
     return {
-      render: this.onChildRender
```

Comment:
I think I actually prefer not adding the "ed", but that could be a matter of opinion..  I suppose it was already this way in the CollectionView docs, but I try and stick to before prefixes and all of my after events are present tense to match how the events are written.